### ISSUE TITLE
another core around jemalloc

### ISSUE THREAD

**Issue by h0lzi** [joyent/illumos-joyent#36]
on
joyent_20150305T101223Z

I tested plex again. It cored again. around jemalloc as it seems. (It did work a few build ago, after the jemalloc fix, or it's now harder to trigger)

libjemalloc.so.1`ifree+0x1ed3()

libjemalloc.so.1`free+0x1f()

libcurl.so.4`Curl_close+0x83()

libjemalloc.so.1`malloc()

0xf87d8348f87d8948()

core can be found here:
https://buildfarm.opencsw.org/~jh/core.Plex.Media.Serve.19704

**Comment by lloydde** [comment]
@h0lzi we can't access https://buildfarm.opencsw.org/~jh/core.Plex.Media.Serve.19704 , could you fix the permissions?

```
% curl https://buildfarm.opencsw.org/\~jh/core.Plex.Media.Serve.19704
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>403 Forbidden</title>
</head><body>
<h1>Forbidden</h1>
<p>You don't have permission to access /~jh/core.Plex.Media.Serve.19704
on this server.</p>
</body></html>
```

Same result in browser.

**Comment by h0lzi** [comment]
ups sorry.
fixed

**Comment by lloydde** [comment]
Thanks @h0lzi , I've reopened https://smartos.org/bugview/OS-3711

**Comment by lloydde** [comment]
@bcantrill confirmed this is not a dup of OS-3711, so I've opened https://smartos.org/bugview/OS-4005 .

**Comment by h0lzi** [comment]
@lloydde @bcantrill I was not able to reproduce this crash again. Looks more like a one time event. So I would close this bug for now I guess. As @bcantrill pointed out Plex "killed" itself. So maybe there is some logic in Plex to do that and it's fine :)

**Comment by lloydde** [comment]
@h0lzi thnx for the update.

**Comment by RSully** [comment]
Fixed in a9a4526?

**Comment by h0lzi** [comment]
@RSully yes that should fix that.
